tests/conftest.py

```diff
@@ -0,0 +1,342 @@
+"""
+This conftest file contains simple nodes (nodes with minimal required arguments)
+and complex node (nodes that have all possible arguments), to use for testing.
+
+Since nodes often depend on other nodes copying and pasting nodes is not ideal,
+and keeping all nodes in one file makes it easier/cleaner to create tests.
+
+The fixtures are all functional fixtures that stay consistent between all tests.
+"""
+
+import pytest
+
+import cript
+
+
+@pytest.fixture(scope="session")
+def cript_api():
+    """
+    creates a CRIPT API object, used for integration tests for all nodes
+
+    * saving
+    * getting
+    * updating
+    * deleting
+
+    Returns
+    -------
+    cript.API
+        api object used to interact with CRIPT
+    """
+    return cript.API(host="https://cript.org", token="123465")
```

Comment:
I am not doing anything with the cript_api yet because it is just a place holder, we can come back to it when we are working on api tests

also see [my other comment about yields vs return fixtures](https://github.com/C-Accel-CRIPT/Python-SDK/pull/52#discussion_r1160084042)

Reply:
For validation that won't work.
If you are not in a context manager, the global cache is `None`.
So yielding from the context manager is required here.